include/bout/solver.hxx

```diff
@@ -290,7 +292,7 @@ class Solver {
   ///
   /// @param[inout] outputfile   The file to add variable to
   /// @param[in] save_repeat    If true, add variables with time dimension
-  void outputVars(Datafile &outputfile, bool save_repeat=true);
+  virtual void outputVars(Datafile &outputfile, bool save_repeat=true);
```

Comment:
Is this a separate bugfix?

Reply:
This change and the changes in `src/solver/impls/power/power.hxx,power.cxx` are introduced in 
6c70779. The purpose is to remove a use of the global `dump` in `PowerSolver` by overriding `Solver::outputVars` instead of using `dump.add(eigenvalue, "eigenvalue", true)`.